src/__tests__/type.js

```diff
@@ -853,12 +853,63 @@ test('typing on input type file should not result in an error', () => {
   return userEvent.type(element, 'bar').catch(e => expect(e).toBeUndefined())
 })
 
-test('should submit a form when ENTER is pressed on input', () => {
+test('should submit a form with only one input when ENTER is pressed on it', () => {
```

Comment:
I think it would make sense to split this test up into three different tests. Is there a reason they're combined?

Reply:
The first time I have made three tests but because they are related to the same thing I have merged them in a single one

Reply:
Often I'm a fan of [fewer, longer tests](https://kentcdodds.com/blog/write-fewer-longer-tests), but in this case I think these make complete sense to be separate tests. Thanks!

Reply:
No problem 🙂